### PR TITLE
Make possible to run PowerAuth Admin as standalone

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: .run/PowerAuthAdminApplication.run.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="PowerAuthAdminApplication" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+    <option name="ACTIVE_PROFILES" />
+    <option name="DEBUG_MODE" value="true" />
+    <module name="powerauth-admin" />
+    <option name="SPRING_BOOT_MAIN_CLASS" value="io.getlime.security.app.admin.PowerAuthAdminApplication" />
+    <option name="VM_PARAMETERS" value="-Dserver.servlet.context-path=/powerauth-admin -Dserver.port=8082" />
+    <option name="WORKING_DIRECTORY" value="file://$MODULE_WORKING_DIR$" />
```
Comment (banterCZ) [inline .run/PowerAuthAdminApplication.run.xml:8]: This one was the most tricky! https://stackoverflow.com/a/57319734/204950

File: powerauth-admin/pom.xml [hunk 2]
```diff
@@ -68,9 +68,10 @@
 
         <!-- JSTL -->
         <dependency>
-            <groupId>org.glassfish.web</groupId>
-            <artifactId>javax.servlet.jsp.jstl</artifactId>
-            <version>1.2.5</version>
+            <groupId>javax.servlet</groupId>
```
Comment (banterCZ) [inline powerauth-admin/pom.xml:71]: We should use jakarta artifacts, but let's keep steps as small as possible.

File: docs-private/Developer-How-To-Start.md [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+
+
+## PowerAuth Admin Server
+
+
+### Standalone Run
+
+- Enable maven profile `standalone`
+- Use IntelliJ Idea run configuration at `../.run/PowerAuthAdminApplication.run.xml`
+- Open [http://localhost:8082/powerauth-admin/actuator/health](http://localhost:8082/powerauth-admin/actuator/health) and you should get `{"status":"UP"}`
```
Comment (banterCZ) [inline docs-private/Developer-How-To-Start.md:21]: See the port **8082** to avoid conflicts.

File: docs-private/Developer-How-To-Start.md [hunk 4]
```diff
@@ -1,9 +1,21 @@
...
+
 ## PowerAuth Java Server
 
+
 ### Standalone Run
 
 - Enable maven profile `standalone`
 - Use IntelliJ Idea run configuration at `../.run/PowerAuthServerApplication.run.xml`
 - Open [http://localhost:8080/powerauth-java-server/actuator/health](http://localhost:8080/powerauth-java-server/actuator/health) and you should get `{"status":"UP"}`
+
```
Comment (banterCZ) [inline docs-private/Developer-How-To-Start.md:12]: Double empty line before the header on purpose.

File: powerauth-admin/pom.xml [hunk 5]
```diff
@@ -68,9 +68,10 @@
 
         <!-- JSTL -->
         <dependency>
-            <groupId>org.glassfish.web</groupId>
```
[the remark this answers is not in the document]
Comment (banterCZ) [reply powerauth-admin/pom.xml:71]: The main reason I change it, is that there is a transitive dependency to servlet 2.5 which causes classloader issues https://mvnrepository.com/artifact/org.glassfish.web/javax.servlet.jsp.jstl/1.2.5

**Review by romanstrobl — APPROVED** [review]
I assume it works, so let's merge this change.

**Pull request merged**